app/views/users/registrations/verification_email_sent.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l">Verify your email</h2>
+    <p>We have sent you a link to verify your email. This will expire in 24 hours.</p>
+    <p>It may be in your spam or junk folder</p>
+    <br>
```

Comment:
There are govuk design classes for spacing, we should use them over brs. 

https://design-system.service.gov.uk/styles/spacing/

`govuk-!-margin-bottom-2` or something like that will do I think.